source/Cosmos.System2_Plugs/System/IO/FileImpl.cs

```diff
@@ -162,11 +162,31 @@ public static void WriteAllBytes(string aFile, byte[] aBytes)
 
         public static void Copy(string srcFile, string destFile)
         {
-            using (var xFS = new FileStream(srcFile, FileMode.Open))
+            try
```

Comment:
The `FileStream` code didn't work? I think it's faster to use `FileStream`.

Reply:
No, it doesn't work.

Reply:
And the file created is empty